src/CoreBundle/Resources/public/javascript/tableEditor.js

```diff
@@ -710,14 +706,99 @@ CHAMELEON.CORE.MTTableEditor.DeleteRecordWithCustomConfirmMessage = function (sC
     if (confirm(sConfirmText)) {
         window.onbeforeunload = function () {
         };
-        PleaseWait();
+        CHAMELEON.CORE.processingDialog('show');
         document.cmseditform.elements['module_fnc[contentmodule]'].value = 'Delete';
         document.cmseditform.submit();
     }
-}
+};
+
+$(document).ready(function () {
+    CHAMELEON.CORE.MTTableEditor.setGetActiveTab();
+    CHAMELEON.CORE.MTTableEditor.inputFields();
+});
 
-function updateIframeSize(sFieldName,iHeight){
+
+CHAMELEON.CORE.MTTableEditor.setGetActiveTab = function () {
+    var url = document.URL;
+    var hash = url.substring(url.indexOf('#'));
+
+    $(".nav-tabs").find("li a").each(function (key, val) {
+        if (hash == $(val).attr('href')) {
+            $(val).click();
+        }
+        $(val).click(function () {
+            location.hash = $(this).attr('href');
+        });
+    });
+};
+
+function updateIframeSize(sFieldName, iHeight) {
     if (sFieldName != '') {
         $('#' + sFieldName + '_iframe').height(iHeight + 'px');
     }
-}
\ No newline at end of file
+}
+
+CHAMELEON.CORE.MTTableEditor.inputFields = function () {
+    $('[data-datetimepicker-option]').each(function () {
+        $(this).datetimepicker($(this).data("datetimepicker-option"));
+    });
+
+    $('.datetimepicker-input').each(function () {
+        var id = $(this).attr('id');
+
+        // This custom-event of the datetimepicker only works with the id of the element.
+        $('#' + id).on("change.datetimepicker", function (e) {
+            var moment = e.date;
+
+            if (moment !== undefined) {
```

Comment:
Cannot explain exactly what issue we had here. It was something with destroying the date object on close or something. Will not touch this if you are o.k. with it, because this had costs us hours of debugging and ranting about the date library and this way it works very well. :-)
But i added an early return.